lavamoat/browserify/beta/policy.json

```diff
@@ -686,18 +686,10 @@
     },
     "@metamask/announcement-controller": {
       "packages": {
-        "@metamask/announcement-controller>@metamask/base-controller": true
-      }
-    },
-    "@metamask/announcement-controller>@metamask/base-controller": {
-      "packages": {
-        "immer": true
```

Comment:
Are we no longer using Immer on the controller side?

Reply:
We still are, but we removed a copy of the base controller. Previously we had two different versions, now we have one fewer. That's all that's being removed here.